### PR TITLE
Reorganized docs for docusaurus publish

### PR DESCRIPTION
**Pull request by writinwaters**
### What problem does this PR solve?

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

### Type of change

- [x] Documentation Update

### CODE REVIEW

**Review by KevinHuSh — APPROVED** [review]

File: docs/_category_.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Get Started",
+  "position": 1,
+  "link": {
+    "type": "generated-index",
+    "description": "RAGFlow Quick Start"
```
Comment (JinHai-CN) [inline docs/_category_.json:6]: low case?

File: docs/guides/llm_api_key_setup.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-<div align="center" style="margin-top:20px;margin-bottom:20px;">
-<img src="https://github.com/infiniflow/ragflow/assets/12318111/e4e4066c-e964-45ff-bd56-c3fc7fb18bd3" width="1000"/>
-</div>
-
+---
+sidebar_position: 4
+slug: /llm_api_key_setup
+---
+
+# Set Before Starting The System
```
Comment (JinHai-CN) [inline docs/guides/llm_api_key_setup.md]: This title need to update.

File: docs/guides/manage_files.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+---
+sidebar_position: 3
+slug: /manage_files
+---
+
 #  Manage files
```
Comment (JinHai-CN) [inline docs/guides/manage_files.md:6]: File management?

File: docs/guides/max_map_count.md [hunk 4]
```diff
@@ -1,66 +1,71 @@
...
-```
-and appending a line which reads:
-```bash
-vm.max_map_count = 262144
+---
+sidebar_position: 7
+slug: /max_map_count
+---
+
+# Update vm.max_map_count
```
Comment (JinHai-CN) [inline docs/guides/max_map_count.md:6]: This file need to merge into Get started.

File: docs/quickstart.md [hunk 5]
```diff
@@ -49,7 +54,7 @@ This quick start guide describes a general process from:
 
 3. Build the pre-built Docker images and start up the server:
 
-   > Running the following commands automatically downloads the *dev* version RAGFlow Docker image. To download and run a specified Docker version, update `RAGFLOW_VERSION` in **docker/.env** to the intended version, for example `RAGFLOW_VERSION=v0.6.0`, before running the following commands.
+   > Running the following commands automatically downloads the *dev* version RAGFlow Docker image. To download and run a specified Docker version, update `RAGFLOW_VERSION` in **docker/.env** to the intended version, for example `RAGFLOW_VERSION=v0.5.0`, before running the following commands.
```
Comment (JinHai-CN) [inline docs/quickstart.md]: Why change version?

File: docs/references/CONTRIBUTING.md [hunk 6]
```diff
@@ -1,52 +1,52 @@
...
-- Refer to a corresponding GitHub issue in your PR description if applicable. 
-- Include sufficient design details for *breaking changes* or *API changes* in your description.
-
-### Reviewing & merging a PR
+---
+sidebar_position: 0
+slug: /contribution_guidelines
+---
+
+# Contribution Guidelines
```
Comment (JinHai-CN) [inline docs/references/CONTRIBUTING.md]: Contribution guidelines

File: docs/references/faq.md [hunk 7]
```diff
@@ -1,3 +1,8 @@
+---
+sidebar_position: 3
+slug: /faq
+---
+
 # Frequently Asked Questions
```
Comment (JinHai-CN) [inline docs/references/faq.md]: Frequently asked questions

File: docs/quickstart.md [hunk 8]
```diff
@@ -1,3 +1,8 @@
+---
+sidebar_position: 1
+slug: /
+---
+
 # Quickstart
```
Comment (JinHai-CN) [inline docs/quickstart.md]: Quick start

File: docs/guides/llm_api_key_setup.md [hunk 9]
```diff
@@ -1,19 +1,21 @@
...
+
+In **user_default_llm** of [service_conf.yaml](./docker/service_conf.yaml), you need to specify LLM factory and your own _API_KEY_. 
+RagFlow supports the flowing LLM factory, and with more coming in the pipeline:
+
+> [OpenAI](https://platform.openai.com/login?launch), [Tongyi-Qianwen](https://dashscope.console.aliyun.com/model), 
+> [ZHIPU-AI](https://open.bigmodel.cn/), [Moonshot](https://platform.moonshot.cn/docs)
+
+After sign in these LLM suppliers, create your own API-Key, they all have a certain amount of free quota.
+
+## After Starting The System
```
Comment (JinHai-CN) [inline docs/guides/llm_api_key_setup.md:20]: A better title ?

**Pull request merged**